pkg/cloudinit/layout_test.go

```diff
@@ -251,6 +266,9 @@ BYT;
 				Layout: layout,
 			}
 			runner.SideEffect = func(command string, args ...string) ([]byte, error) {
+				if command == "lsblk" && args[0] == "--list" {
```

Comment:
not a hard request but IMO it would be nice to make all the repeated control flows functions, so in the future they only need to get changed in one place